gax-java/dependencies.properties

```diff
@@ -64,7 +64,7 @@ maven.com_google_j2objc_j2objc_annotations=com.google.j2objc:j2objc-annotations:
 maven.com_google_auto_value_auto_value=com.google.auto.value:auto-value:1.10.1
 maven.com_google_auto_value_auto_value_annotations=com.google.auto.value:auto-value-annotations:1.10.1
 maven.com_google_api_api_common=com.google.api:api-common:2.3.1
-maven.org_threeten_threetenbp=org.threeten:threetenbp:1.6.4
+maven.org_threeten_threetenbp=org.threeten:threetenbp:1.6.5
```

Comment:
This matches https://github.com/googleapis/gapic-generator-java/blob/0ccf457a774bd3dbe3b101d82222bdc39f6e5b0e/gax-java/pom.xml#L66